### PR TITLE
Update README.md and Fix broken image link

### LINKED ISSUE #5: Readme has broken image links
This repo's [readme.md](https://github.com/layer5io/gowrk2/blob/master/README.md) is missing a couple images that need to be copied over to this repo.

### PR DESCRIPTION
**Pull request by Adeola-Adesoba**
Signed-off-by: Adeola 46505882+Adeola-Adesoba@users.noreply.github.com

This fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ go run main.go
 <h2><a name="contributing"></a><a name="community"></a> <a href="http://slack.layer5.io">Community</a> and <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">Contributing</a></h2>
 Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>! Contributors are expected to adhere to the <a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Code of Conduct</a>.
 
-<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="img/readme/slack-128.png" style="margin-left:10px;padding-top:5px;" width="110px" align="right" /></a>
+<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src=".github/readme/images//slack-128.png" style="margin-left:10px;padding-top:5px;" width="110px" align="right" /></a>
```
Comment (hinupurthakur) [inline README.md]: `src` needs to be updated with with valid path of the image.
e.g. create a folder img/readme and add the image of slack, something like this https://github.com/layer5io/meshery/blob/master/docs/assets/img/readme/slack-128.png

File: README.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ go run main.go
...
 
-<a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src="img/readme/community.svg" style="margin-right:8px;padding-top:5px;" width="140px" align="left" /></a>
+<a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src=".github/readme/images//community.svg" style="margin-right:8px;padding-top:5px;" width="140px" align="left" /></a>
```
Comment (hinupurthakur) [inline README.md]: Similiarly, image path needs to be updated with the image at that location.
e.g. https://github.com/layer5io/meshery/blob/master/docs/assets/img/readme/community.png

**Review by hinupurthakur — CHANGES_REQUESTED** [review]

**Comment by hinupurthakur** [comment]
Also, @Adeola-Adesoba Are you using the correct email to sign-off your commits?

**Comment by Adeola-Adesoba** [comment]
> Also, @Adeola-Adesoba Are you using the correct email to sign-off your commits?

Sure, the DCO check passed. Did you notice something?

File: README.md (hunk 1 above)
Comment (Adeola-Adesoba) [reply README.md]: Thanks! I will make the necessary changes

**Comment by hinupurthakur** [comment]
> > Also, @Adeola-Adesoba Are you using the correct email to sign-off your commits?
> 
> Sure, the DCO check passed. Did you notice something?

@Adeola-Adesoba No no, I am not saying about DCO check. You are using 46505882+Adeola-Adesoba@users.noreply.github.com as your email to sign the commits, to me it doesn't feel like a correct email so just asked about that.

**Review by hinupurthakur — COMMENTED** [review]
@leecalcote Should we create a new folder in gowrk2 repository for images that are to be used in readme or is it fine if we use it from Meshery repository as @Adeola-Adesoba have done here?

**Comment by Adeola-Adesoba** [comment]
> > > Also, @Adeola-Adesoba Are you using the correct email to sign-off your commits?
> > 
> > 
> > Sure, the DCO check passed. Did you notice something?
> 
> @Adeola-Adesoba No no, I am not saying about DCO check. You are using [46505882+Adeola-Adesoba@users.noreply.github.com](mailto:46505882+Adeola-Adesoba@users.noreply.github.com) as your email to sign the commits, to me it doesn't feel like a correct email so just asked about that

**Comment by Nikhil-Ladha** [comment]
I think it's better to create a new folder here, and then use that image. It will be a better way of working. So, @Adeola-Adesoba if you could make that change then we will be good to go with this PR.

**Comment by Adeola-Adesoba** [comment]
@NupurThakur27  @Nikhil-Ladha  can you merge the images folder PR so that I can use the images there now?

**Comment by Nikhil-Ladha** [comment]
@Adeola-Adesoba you can use the image from the images folder that you are adding even now, while making the PR, it will get recognised. Just one thing can you make the name of the folder to all small letters like `images` instead of ` Images`..

**Comment by Adeola-Adesoba** [comment]
@NupurThakur27 @Nikhil-Ladha  I deleted this PR and created a new one. You should have gotten the notification, not sure.

**Pull request closed without merging**